static/scss/user_menu.scss

```diff
@@ -20,27 +20,34 @@
   .user-menu-dropdown.open {
     display: flex;
     flex-direction: column;
-    background-color: white;
+    background-color: rgba(0,0,0,.73);
     position: absolute;
     width: 80%;
     top: 100%;
     left: 10%;
-    border-radius: 3px;
+    border-radius: 2px;
     box-shadow: 0 0 10px grey;
+    z-index: 100;
   }
 
   .user-menu-dropdown > a {
-    margin: 5px 10px;
-    color: $font-black;
-    opacity: .87;
+    margin: 6px 13px;
+    color: rgba(255,255,255,.8);
+    font-weight: 300;
+  }
+
+  .user-menu-dropdown > a:hover {
```

Comment:
Feel free to leave this but you can also nest these kinds of directives:

    .user-menu-dropdown {
      > a {
        ...
      }
      
      > a:hover {
        ...
      }